execution_workflows/IsoSCM/bin/postprocess.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+
+import sys
+import argparse
+import csv
+
+def parse_args(args=None):
+    Description = "Reformat IsoSCM bed file into the output file of identification"
+    Epilog = "Example usage: python postprcoess.py <FILE_IN> <RUN_IDENTIFICATION> <IDENTIFICATION_FILE_OUT>"
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument("FILE_IN", help="Input IsoSCM output bed file.")
+    parser.add_argument("RUN_MODE", help="Run mode could be 'identification'")
+    parser.add_argument("FILE_OUT", help="Name of output file")
+    return parser.parse_args(args)
+
+
+def parse_input_file(file_in):
+    """
+    This function parses IsoSCM output file to file for
+    identification or relative usage quantification challenge
+    :param file_in: IsoSCM compare step output file to be reformatted to relative usage quantification bed file
+    :return: N/A
+    """
+    outputs = []
+    with open(file_in) as fd:
+        rd = csv.reader(fd, delimiter="\t", quotechar='"')
+        next(fd, None)
+        for row in rd:
+            # obtain proximal PAS
+            chrom = row[2].split(":")[0]
+            # need to do -1 to convert from 1-based gtf file coordinate to 0-based bed file coordinate
+            start = int(row[2].split(":")[1].split("-")[1]) - 1
+            end = start + 1
+            strand = row[8]
+            name = '|'.join([chrom, str(start) + ":" + str(end), strand])
```

Comment:
Small comment, but not worth blocking this for now as it is more important for relative usage quantification output:

For a tool like this we will need to keep track of which pairs of PAS go together when we match and correlate dPAS versus pPAS values. Because this tool only outputs coordinates (no gene or transcript info), it would be helpful to have the `name` here be unique to the pair of PAS that are derived from each line of this file. 

I was thinking the `locus` information (`row[7]`), but that is not unique in the mouse cortex dataset we ran so we might have to use that plus a line number or something. I'm thinking it should end up looking something like: 

[line_number]_[locus]_proximal and [line_number]_[locus]_distal

Again, not 100% necessary for this so feel free to ignore for now.

Reply:
Ah yes sure thing!! I'll make sure to incorporate this in the relative quant isoscm PR! :D